### PR TITLE
New package: K1JT.wsjtx version v2.6.1

### PR DESCRIPTION
**Pull request by xylo04**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111637&drop=dogfoodAlpha

### CODE REVIEW

**Comment by xylo04** [comment]
@microsoft-github-policy-service agree

File: manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1
+PackageLocale: en-US
+Publisher: Joe Taylor, K1JT
+PackageName: wsjtx
```
Comment (Trenly) [inline manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.locale.en-US.yaml:8]:
```suggestion
PackageName: WSJT-X: Digital Modes for Weak Signal Communications in Amateur Radio
```
This should match control panel

File: manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1
```
Comment (Trenly) [inline manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.6.1
```
This should match control panel. After changing this value, the name of the folder will need to be updated.

File: manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1
```
Comment (Trenly) [inline manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml:5]:
```suggestion
PackageVersion: 2.6.1
```
This should match control panel. After changing this value, the name of the folder will need to be updated.

File: manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1
```
Comment (Trenly) [inline manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.yaml:5]:
```suggestion
PackageVersion: 2.6.1
```
This should match control panel. After changing this value, the name of the folder will need to be updated.

File: manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1
+Installers:
+- InstallerUrl: https://wsjt.sourceforge.io/downloads/wsjtx-2.6.1-win64.exe
```
Comment (Trenly) [inline manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml:7]: The x86 installer should also be added - 
> https://wsjt.sourceforge.io/downloads/wsjtx-2.6.1-win32.exe

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml (hunk 5 above)
Comment (xylo04) [reply manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml]: Done

**Comment by xylo04** [comment]
Thanks for the thorough review! Please take another look.

File: manifests/k/K1JT/wsjtx/2.6.1/K1JT.wsjtx.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: 2.6.1
+PackageLocale: en-US
+Publisher: Joe Taylor, K1JT
+PackageName: WSJT-X: Digital Modes for Weak Signal Communications in Amateur Radio
```
Comment (Trenly) [inline manifests/k/K1JT/wsjtx/2.6.1/K1JT.wsjtx.locale.en-US.yaml:8]:
```suggestion
PackageName: WSJT-X Digital Modes for Weak Signal Communications in Amateur Radio
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**